main.go

```diff
@@ -30,14 +31,18 @@ func main() {
 	}
 	var address string
 	var certPaths string
+	var namespaceWhitelist string
 	var tokenPath string
 	var vaultURL string
 	var help bool
+	var monitorSecrets bool
 	flag.StringVar(&address, "address", ":9005", "address which cert-exporter uses to listen and serve")
 	flag.StringVar(&certPaths, "cert-paths", "", "comma separated folders containing certs to export")
+	flag.StringVar(&namespaceWhitelist, "namespace-whitelist", "", "comma separated namespaces in which to monitor TLS secrets")
```

Comment:
I thought it made sense to give the option of not scraping secrets in all namespaces as this will run in TCs too

Reply:
makes sense to me, but would it be maybe better to call it simply "--namespaces"? If there is a "whitelist" I'd expect the presence of a "blacklist" also.